aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -94,6 +94,24 @@ std::tuple<at::Tensor,at::Tensor,at::Tensor> cudnn_convolution_transpose_backwar
 #include <stdint.h>
 #include <unordered_map>
 
+// Note [behavior of cudnnFind and cudnnGet]
+// You'll notice that by default, in the ConvolutionDescriptor, we do the following:
+//
+//     AT_CUDNN_CHECK(cudnnSetConvolutionMathType(mut_desc(), CUDNN_DEFAULT_MATH));
+//     if(dataType == CUDNN_DATA_HALF)
+//       AT_CUDNN_CHECK(cudnnSetConvolutionMathType(mut_desc(), CUDNN_TENSOR_OP_MATH));
+//
+// When cudnnSetConvolutionMathType is called before cudnnGet/cudnnFind, it informs 
+// cudnnGet/cudnnFind to iterate/take into account both tensor core and non-tensor-core algos.
+// If you don't call cudnnSetConvolutionMathType before calling cudnnGet/cudnnFind,
+// cudnnGet/cudnnFind may not pick tensor core algos.
```

Comment:
I'm planning to accept this patch as is, but one thing I don't quite get from the description here is whether or not the `MATH` parameter you pass in matters or not.

Reply:
It does matter as it turns out. Should I word this differently (I don't think one can decipher this from the cuDNN docs either)? For instance, before this patch, since I didn't pass CUDNN_TENSOR_OP_MATH, cudnnFind/cudnnGet was not picking the algorithms that enable tensor cores.